content/kots-cli/upstream/download.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2022-02-24
+linktitle: "download"
+title: kots upstream download
+weight: 90182
+---
+
+The `kots upstream download` command retries downloading a failed update of the upstream application.
+
+> Introduced in KOTS v1.65.0
+
+### Usage
+```bash
+kubectl kots upstream download [app-slug] [flags]
+```
+* _Replace `[app-slug]` with the app slug for your KOTS application (required)._
+* _Provide `[flags]` according to the table below._
+
+| Flag                              | Type   | Description                                                                                      |
+|:----------------------------------|--------|--------------------------------------------------------------------------------------------------|
+| `-h, --help`                      |        | help for upstream download                                                                       |
+| `--kubeconfig`                    | string | the kubeconfig to use _(default is `$KUBECONFIG`. If unset, then `$HOME/.kube/config`)_          |
+| `-n, --namespace`                 | string | the namespace where the admin console is running _(required)_                                    |
+| `--sequence`                      | int    | The local app sequence for the version to retry downloading _(required)_                         |
```

Comment:
Will `upstream upgrade` print out the sequence? So that if it errors, someone can get the sequence and bring it right over to this `download` command.

Reply:
Good question, it will not, but we can make it do that.

Reply:
I don't want to bloat the feature, but I'm concerned `download` is useless if the sequence isn't available. Something to think about. Maybe there's another way it's available, but in the `upstream upgrade` output would be the best UX since they probably go to `download` after running `upgrade`.

Reply:
Decided in standup to leave this is as since other commands require the sequence and provide no way to know what it is either. It's a bad experience, but it's no different than other commands at this point.